moveit_simple/test/moveit_simple.cpp

```diff
@@ -419,4 +435,49 @@ TEST(MoveitSimpleTest, kinematics)
 
 }
 
+
+TEST(MoveitSimpleTest, copy_current_pose)
+{
+  moveit_simple::Robot robot(ros::NodeHandle(), "robot_description", "manipulator");
```

Comment:
This test and others has a lot of the same setup.  Look into Gtest Fixtures to eliminate some of this common code.